eth/backend.go

```diff
@@ -197,18 +147,48 @@ func (b *backend) retryRemoteCall(remoteCall func() ([]byte, error)) (out []byte
 	return out, err
 }
 
-func makeABIMap() (map[string]*abi.ABI, error) {
+func makeABIMap() map[string]*abi.ABI {
 	abiMap := make(map[string]*abi.ABI)
 
 	for _, ABI := range abis {
 		parsedAbi, err := abi.JSON(strings.NewReader(ABI))
 		if err != nil {
-			return map[string]*abi.ABI{}, err
+			glog.Errorf("Error creating ABI mappings, can not parse transaction logs err=%v", err)
```

Comment:
```suggestion
			glog.Errorf("Error creating ABI map err=%v", err)
```

since there is no transaction log parsing in this function.